.github/pull_request_template.md

```diff
@@ -13,12 +13,12 @@ Please explain **IN DETAIL** what the changes are in this PR and why they are ne
 Please **NOTE** that:
 - Do not assume reviewers understand the original issue
 
-## What are the type of the changes? (mandatory)
+## What are the type of changes? (mandatory)
 
 The currently defined types are listed below, please pick one of the types for this PR by removing the others:
-- New feature (change which adds functionality)
-- Improvement (change which is an improvement to an existing feature)
-- Bug fix (change which fixes an issue)
+- New feature (a change which adds functionality)
```

Comment:
How many changes? One or many? In title it is "changes" but here it is "a change".

Reply:
Maybe change the title to "What is the type of the change?"